client/scss/settings/_variables.scss

```diff
@@ -28,9 +28,10 @@ $breakpoints: (
 );
 
 // colours
-$color-teal: #007d7e;
-$color-teal-darker: darken(adjust-hue($color-teal, 1), 4);
-$color-teal-dark: darken(adjust-hue($color-teal, 1), 7);
+$color-teal: var(--color-primary);
+$color-teal-darker: var(--color-primary-darker);
+$color-teal-dark: var(--color-primary-dark);
+$color-teal-darkest: var(--color-primary-darkest);
```

Comment:
I would welcome a follow-up PR to search-and-replace all the Sass variables so they switch to `color-primary` as well. But glad this isn’t in the same PR for now!